Cargo.toml

```diff
@@ -19,7 +19,7 @@ web-sys = ["web_sys", "js-sys", "wasm-bindgen"]
 stdweb = ["std_web", "webgl_stdweb"]
 
 [target.'cfg(not(any(target_arch = "wasm32")))'.dependencies]
-glutin = { version = "0.22.0-alpha3", optional = true }
+glutin = { version = "0.22.0-alpha5", optional = true }
```

Comment:
Could we use `version = "=0.22.0-alpha5"` for both of these version strings (in order to prevent automatically updating to latest alphas)?